machines/issuersMachine.ts

```diff
@@ -153,11 +224,20 @@ export const IssuersMachine = model.createMachine(
           src: 'generateKeyPair',
           onDone: [
             {
-              actions: ['setPublicKey', 'setPrivateKey', 'storeKeyPair'],
+              actions: [
+                'setPublicKey',
+                'setLoadingReasonAsDownloadingCreds',
```

Comment:
please remove the abbreviation  "creds" and use full name for more clarity "credential"

Reply:
Refactored